Can't build image for ARM

Hello,
I have a probleme when i build image on raspberry.
I have this error:
```
configure: error: Cannot find ldap libraries in /usr/lib/x86_64-linux-gnu.
Removing intermediate container 98783f8dc2c0
The command '/bin/sh -c docker-php-ext-configure mysqli --with-mysqli=mysqlnd &&     docker-php-ext-install mysqli &&     docker-php-ext-configure gd --with-freetype-dir=/usr/include/freetype2 --with-png-dir=/usr/include --with-jpeg-dir=/usr/include &&     docker-php-ext-install gd &&     docker-php-ext-install curl &&     docker-php-ext-install json &&     docker-php-ext-install snmp &&     docker-php-ext-install sockets &&     docker-php-ext-install pdo_mysql &&     docker-php-ext-install gettext &&     ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h &&     docker-php-ext-configure gmp --with-gmp=/usr/include/x86_64-linux-gnu &&     docker-php-ext-install gmp &&     docker-php-ext-install pcntl &&     docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu &&     docker-php-ext-install ldap &&     pecl install mcrypt-1.0.1 &&     docker-php-ext-enable mcrypt &&     echo ". /etc/environment" >> /etc/apache2/envvars &&     a2enmod rewrite' returned a non-zero code: 1
```
Can you help me with this problem or publish an image for ARM ?
Thanks a lot.

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,11 +34,11 @@ RUN docker-php-ext-configure mysqli --with-mysqli=mysqlnd && \
     docker-php-ext-install sockets && \
     docker-php-ext-install pdo_mysql && \
     docker-php-ext-install gettext && \
-    ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h && \
-    docker-php-ext-configure gmp --with-gmp=/usr/include/x86_64-linux-gnu && \
+    ln -s /usr/include/$(uname -m)-linux-gnu/gmp.h /usr/include/gmp.h && \
+    docker-php-ext-configure gmp --with-gmp=/usr/include/$(uname -m)-linux-gnu && \
     docker-php-ext-install gmp && \
     docker-php-ext-install pcntl && \
-    docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu && \
+    docker-php-ext-configure ldap --with-libdir=lib/$(uname -m)-linux-gnu && \
     docker-php-ext-install ldap && \
     pecl install mcrypt-1.0.1 && \
     docker-php-ext-enable mcrypt && \

